### DISCUSSION TITLE
windows下的安装教程视频版

### DISCUSSION THREAD

**Discussion by borntree** [HibiKier/zhenxun_bot#1805] [Ideas]
原来用的酷播云，可以在线看，但是免费用户12个G用完就没了，300M的教学视频也就够40个人看，所以视频传到微云了

视频:https://share.weiyun.com/zcpZsqyu

参考文档:https://hibikier.github.io/zhenxun_bot/docs/installation_doc/

所需文件:https://share.weiyun.com/8Bl5uqSr 
微云下载限速的话，自己存自己号上一份，然后QQ发送文件，选择微云，找到这个文件并发送，在QQ下载就行了。

**Comment by wlz-remake** [comment]
呃视频在哪？（顺便说一声辛苦了

**Comment by cywy321123** [comment]
同求（紧贴大佬）

**Comment by borntree** [comment]
> 呃视频在哪？（顺便说一声辛苦了

忘了写了，今天下午有点急事

**Comment by borntree** [comment]
> 同求（紧贴大佬）

已更

**Comment by borntree** [comment]
@HibiKier 大佬看看哪里需要修改吗

**Comment by HibiKier** [comment]
![图片](https://user-images.githubusercontent.com/45528451/171178493-69f41e2f-4eda-431a-800c-449a4d162da5.png)
这个可以直接取消捏，总体没啥问题

**Comment by borntree** [comment]
> ![图片](https://user-images.githubusercontent.com/45528451/171178493-69f41e2f-4eda-431a-800c-449a4d162da5.png)
> 这个可以直接取消捏，总体没啥问题

哈哈哈哈，这个我第一次安装这软件时候看的教程没有这一步，成功安装了。
第二次看了另一个教程，有这一步，当时就有点纳闷为啥要多出来这一步，后来想想确实没啥用。

**Comment by KIRING10032** [comment]
大佬辛苦了，视频我打不开能分享个连接给我下载吗，感谢

**Comment by borntree** [comment]
> 大佬辛苦了，视频我打不开能分享个连接给我下载吗，感谢

能打开啊，你看看是不是网络问题，如果还不行的话，我今天把视频传微云上面再分享一遍，现在刚吃完饭得去做个核酸

**Comment by KIRING10032** [comment]
> > 大佬辛苦了，视频我打不开能分享个连接给我下载吗，感谢
> 
> 能打开啊，你看看是不是网络问题，如果还不行的话，我今天把视频传微云上面再分享一遍，现在刚吃完饭得去做个核酸

视频是我校园网的问题，现在已经解决了，谢谢

**Comment by 17705825206** [comment]
大佬视频打不开了
可以给份视频连接吗

**Comment by borntree** [comment]
> 大佬视频打不开了
> 可以给份视频连接吗
> 

我等会传微云一个

**Comment by 17705825206** [comment]
好的

**Comment by borntree** [comment]
> 好的
> 

行了

**Comment by Cintser** [comment]
请问大佬扫码扫不出来怎么办

**Comment by borntree** [comment]
> 请问大佬扫码扫不出来怎么办

我是第一次成功扫码的文件缓存都转移到服务器上面了。他这个安全识别我也不是很清楚怎么解决，具体要去gchttp那边寻求帮助。

**Comment by shinianj** [comment]
在此补充有关pgAdmin4使用数据库的一些知识（既然这里没人说，应该只是特别几个人会出这种问题，尤其是群里出现了这种小白卡这里卡了1天，但教程里面确实没说清楚）：创建角色这里，需要注意，第二栏的Definition，记得输入密码（password）
![_PE@@{5AQ846GFS9K4AW8`7](https://user-images.githubusercontent.com/105840558/176883531-1a33d5d3-3a9d-4545-aaaa-8cd0d45b3a16.png)
然后重点的重点！第三栏privileges，把以下我开了的全部打开！！（除了最后一栏确实不用开，但是第二栏打开会直接打开第三栏和第四栏）
![$)N% L(3XVV469{APB7{7WB](https://user-images.githubusercontent.com/105840558/176884423-3417448f-0eb4-4f88-854e-90ed04db37f7.png)
否则：
第一栏不开：成为用户组，报错：数据库链接失败
第二栏不开：无权限使用表goods_info
第三栏不开：无权限使用表goods_info
第四栏不开：没什么问题，不会报错
最后，我希望各位去用文档里面使用SQL shell的方法创建用户和库（就没有这么多麻烦又有可能会出事的操作了，你还能复制粘贴，你们不开心吗？）

**Comment by Panjqy** [comment]
请问大佬，我看你的视频中是因为媒体基础没安装才出现这种问题，但我自查了一下我媒体基础已经安装了还是出现这个问题是为什么啊
![image](https://user-images.githubusercontent.com/71705912/177038227-90a35b36-ebdf-43f9-b3af-40986cdb9729.png)

**Comment by borntree** [comment]
> 请问大佬，我看你的视频中是因为媒体基础没安装才出现这种问题，但我自查了一下我媒体基础已经安装了还是出现这个问题是为什么啊
> ![image](https://user-images.githubusercontent.com/71705912/177038227-90a35b36-ebdf-43f9-b3af-40986cdb9729.png)
> 
> 

我也不到啊

**Comment by ChrisYan6** [comment]
> 请问大佬，我看你的视频中是因为媒体基础没安装才出现这种问题，但我自查了一下我媒体基础已经安装了还是出现这个问题是为什么啊 ![image](https://user-images.githubusercontent.com/71705912/177038227-90a35b36-ebdf-43f9-b3af-40986cdb9729.png)

我今天也碰到了这问题，好奇怪，明明跟着步骤安装了依赖，但CV2就是没装上，python3.9.12。正在找问题中

**Comment by ChrisYan6** [comment]
> 请问大佬，我看你的视频中是因为媒体基础没安装才出现这种问题，但我自查了一下我媒体基础已经安装了还是出现这个问题是为什么啊 ![image](https://user-images.githubusercontent.com/71705912/177038227-90a35b36-ebdf-43f9-b3af-40986cdb9729.png)

把安装到site-packages目录下cv2目录移动到Lib目录下就行！这应该是CV2的bug，跟真寻没关系。。

**Comment by shinianj** [comment]
> 大佬第一次能运行python bot.py 重启服务器后，再运行这个命令就报错了 怎么解决呀 ![z](https://user-images.githubusercontent.com/107041728/183292590-2a675636-760a-4df1-8c6e-2a1cde1b2e7d.png)

疑似未进虚拟环境，检查启动步骤

**Comment by NoelleDLL** [comment]
> > 大佬第一次能运行python bot.py 重启服务器后，再运行这个命令就报错了 怎么解决呀 ![z](https://user-images.githubusercontent.com/107041728/183292590-2a675636-760a-4df1-8c6e-2a1cde1b2e7d.png)
> 
> 疑似未进虚拟环境，检查启动步骤

没问题了，又重新安装了环境，不知道为什么重启服务器之后之前配的环境就消失了，又重新配了一遍就好了。。。

**Comment by jianjiandeguigui** [comment]
![U`LV~XP2Z2E`$WHH@F@0JBN](https://user-images.githubusercontent.com/108204595/187854825-69913cab-e94f-4bd2-a948-40e147f951c3.png)
![G PFYV3J 2%Y188GEPSXFJ](https://user-images.githubusercontent.com/108204595/187854848-b238302a-2612-481f-b8eb-a4efc823dac3.png)
![MGD0XBUCK3}5QA$G4WMRV%C](https://user-images.githubusercontent.com/108204595/187854859-80985d38-d096-4349-a6dd-837d37f5e43e.png)
大佬能帮我看一下这是什么问题么

**Comment by shinianj** [comment]
> ![ULV~XP2Z2E$WHH@F@0JBN](https://user-images.githubusercontent.com/108204595/187854825-69913cab-e94f-4bd2-a948-40e147f951c3.png) ![G PFYV3J 2%Y188GEPSXFJ](https://user-images.githubusercontent.com/108204595/187854848-b238302a-2612-481f-b8eb-a4efc823dac3.png) ![MGD0XBUCK3}5QA$G4WMRV%C](https://user-images.githubusercontent.com/108204595/187854859-80985d38-d096-4349-a6dd-837d37f5e43e.png) 大佬能帮我看一下这是什么问题么

数据库连接错误，自己检查configs/config.py中的配置是否正确

**Comment by jianjiandeguigui** [comment]
![BC@~HB{MD E70({}CGMHA$V](https://user-images.githubusercontent.com/108204595/187923843-edd7725c-28d6-40f8-a6c7-6c818c90ded6.png)
我的配置和你的文档一模一样跟着视频一步步走的，还是不行

**Comment by shinianj** [comment]
> 

检查自己是否创建了名为testdb的数据库，并保证用户名称和密码是否错误

**Comment by RinTenicol** [comment]
api冒号后面有个空格是真没想到 感谢大佬

**Comment by xuhanlia** [comment]
大佬想问一下，为什么我安装完后没有这个
![image](https://user-images.githubusercontent.com/111414305/188273387-225a6a37-a069-4111-9b15-787ba56c4a8f.png)

**Comment by xuhanlia** [comment]
大佬我打开数据库网页没有这个子界面输密码
![image](https://user-images.githubusercontent.com/111414305/188273883-cb5c29b5-df65-4ea8-a4d7-65d1b6189c12.png)
![image](https://user-images.githubusercontent.com/111414305/188273896-7816391e-3652-4f7f-be5c-5982866bf546.png)

**Comment by shinianj** [comment]
> 大佬我打开数据库网页没有这个子界面输密码 ![image](https://user-images.githubusercontent.com/111414305/188273883-cb5c29b5-df65-4ea8-a4d7-65d1b6189c12.png) ![image](https://user-images.githubusercontent.com/111414305/188273896-7816391e-3652-4f7f-be5c-5982866bf546.png)

我建议你翻译一下，你就知道了

**Comment by gaqwe767** [comment]
![QQ图片20221022095142](https://user-images.githubusercontent.com/64371245/197311391-a79fb08e-5c44-4017-98ac-2aee298ed412.png)
一顿操作结果没法运行，难道不支持win10以上的系统？我在两台电脑试了也是一样的结果，有大佬帮忙看下么....我gocq，oicq，Mirai，都玩过，但我搞不定真寻

**Comment by soxsex** [comment]
请问一下出现下面问题如何解决
![EGL77F~)IM%RM0`70W2N}55](https://user-images.githubusercontent.com/102382267/200148739-ebaf4c57-6d8f-4271-b713-875d3511e2a2.png)

**Comment by shinianj** [comment]
> 请问一下出现下面问题如何解决 ![EGL77F~)IM%RM0`70W2N}55](https://user-images.githubusercontent.com/102382267/200148739-ebaf4c57-6d8f-4271-b713-875d3511e2a2.png)

你bot.py没有运行

**Comment by royuki2431** [comment]
![image](https://user-images.githubusercontent.com/93425616/208295910-2923b2c2-cc06-4693-be55-4a37a65fb6ba.png)
请问一下大佬们，看看这个是什么问题啊。

**Comment by 122506843541** [comment]
到进入虚拟环境这一步的时候卡住了，显示这个是什么原因有办法解决吗
无法加载文件 C:\Users\86137\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-QrKL2Zgd-py3.9\Scripts\activate.ps1，
因为在此系统上禁止运行脚本。有关详细信息，请参阅 https:/go.microsoft.com/fwlink/?LinkID=135170 中的 about_Execution_Pol
icies。
    + CategoryInfo          : SecurityError: (:) []，ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnauthorizedAccess
